docs/content/concepts/partitions-schedules-sensors/partitions.mdx

```diff
@@ -5,19 +5,19 @@ description: Partitioned assets and jobs enable launching backfills, where each
 
 # Partitioned assets and jobs
 
-Partitioning allows a software-defined asset or job to correspond to a set of entities with identical structure but different parameters.
+_Partitioning_ allows a software-defined asset to represent a collection of partitions that can be tracked and materialized independently. In many ways, each partition functions like its own mini-asset, but they all share a common structure and dependencies. Typically, each partition will correspond to a separate file, or a slice of a table in a database.
 
-A _partitioned asset_ is an asset that's composed of a set of partitions, which can be materialized and tracked independently. Most commonly, each partition represents all the records in a data set that fall within a particular time window. Depending on where the asset is stored, each partition might correspond to a file or a slice of a table in a database.
+A common use is for each partition to represent all the records in a data set that fall within a particular time window, e.g. hourly, daily or monthly. Alternatively, each partition can represent a region, a customer, an experiment - any entity that you want to be able to materialize and monitor independently.
 
-A _partitioned job_ is a job where each run corresponds to a partition key. Most commonly, each partition key represents a time window, so, when a job executes, it processes data within one of the time windows.
+A graph of assets with the same partitions implicitly forms a partitioned data pipeline, and you can launch a run that selects multiple assets and materializes the same partition in each asset.
 
-It's common to construct a partitioned job that materializes a particular set of partitioned assets every time it runs.
+Similarly, a _partitioned job_ is a job where each run corresponds to a partition. It's common to construct a partitioned job that materializes a particular set of partitioned assets every time it runs.
```

Comment:
The phrase "materialize a partitioned asset" is potentially confusing-- above, we've just said _a partition_ is an entity that can be "materialized and monitored indepnedently". So what does it mean to materialize a partitioned asset? To materialize _all_ of its partitions?

I recommend: "materializes a particular partition from a set of identically partitioned assets"

Reply:
Good point. Updating.